From 506dc096fcc4a288fc853dfb527d7fa8888dd6f6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bruno=20Mah=C3=A9?= <bruno@cloudera.com>
Date: Fri, 16 Jul 2010 19:51:45 -0700
Subject: [PATCH 0322/1020] CDH-1085. $SYSTEM_LIB_DIR default value shouldn't contain $PREFIX.

Description: $SYSTEM_LIB_DIR default value shouldn't contain $PREFIX.
$PREFIX will be prepended later on
Reason: Bug
Author: Bruno Mahe
Ref: CDH-1085
---
 cloudera/install_hadoop.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cloudera/install_hadoop.sh b/cloudera/install_hadoop.sh
index 68a45e7..e4c39bd 100755
--- a/cloudera/install_hadoop.sh
+++ b/cloudera/install_hadoop.sh
@@ -101,7 +101,7 @@ for var in CLOUDERA_SOURCE_DIR PREFIX BUILD_DIR APACHE_BRANCH; do
 done
 
 LIB_DIR=${LIB_DIR:-$PREFIX/usr/lib/hadoop-$APACHE_BRANCH}
-SYSTEM_LIB_DIR=${SYSTEM_LIB_DIR:-$PREFIX/usr/lib}
+SYSTEM_LIB_DIR=${SYSTEM_LIB_DIR:-/usr/lib}
 BIN_DIR=${BIN_DIR:-$PREFIX/usr/bin}
 DOC_DIR=${DOC_DIR:-$PREFIX/usr/share/doc/hadoop-$APACHE_BRANCH}
 MAN_DIR=${MAN_DIR:-$PREFIX/usr/man}
-- 
1.7.0.4

